cli/command/cli.go

```diff
@@ -125,25 +131,42 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions) error {
 	if err != nil {
 		return err
 	}
+	hasExperimental, err := isEnabled(cli.configFile.ExperimentalCli)
+	if err != nil {
+		return err
```

Comment:
I guess this should `errors.Wrapf(err, "ExperimentalCLI field")` so that the error message mentions which field has a problem